mappings/net/minecraft/entity/EyeOfEnderEntity.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_1672 net/minecraft/entity/EnderEyeEntity
+CLASS net/minecraft/class_1672 net/minecraft/entity/EyeOfEnderEntity
```

Comment:
please no

Reply:
accept mojang inconsistency yog :irritatered:

Reply:
With an anime Minecraft skin, Yoghurt4C's opinion doesn't count

Reply:
why do I have to suffer because of 🇫🇷 terrible frenchmen 🇫🇷